test/verify/check-embed

```diff
@@ -72,6 +74,37 @@ Shell=/shell/index.html
         b.switch_to_frame("embed-auth")
         b.wait_visible("#login-user-input")
 
+    @skipBrowser("Chromium cannot inspect cross-origin frames", "chromium")
+    def testCrossOrigin(self):
+        b = self.browser
+        m = self.machine
+
+        pyhttpd = subprocess.Popen(['python3', '-m', 'http.server', '--bind=localhost',
+                                    '--directory=test/verify/files/embed-cockpit', '12346'])
+        self.addCleanup(pyhttpd.wait)
+        self.addCleanup(pyhttpd.terminate)
```

Comment:
Should these be the other way around? terminate() then wait()?

Reply:
Ahh, cleanup are executed in reverse order!  I propose to make a single cleanup that does everything related to pyhttpd.

Reply:
Correct -- I can do a single one with a `lambda:` instead, which does both things.

Reply:
Done, I added a helper function as `lambda: terminate() or wait()` would look a little too hackish.